packages/cli/core/src/createCli.ts

```diff
@@ -128,8 +128,6 @@ export const createCli = () => {
     ConfigContext.set(loaded.config);
     ResolvedConfigContext.set(resolved);
 
-    await hooksRunner.addRuntimeExports();
```

Comment:
Will this change affect @modern-js/module-tools?

Reply:
I think not..@modern-js/module-tools also run this hook at the start of `prepare`

https://github.com/web-infra-dev/modern.js/blob/main/packages/solutions/module-tools/src/cli.ts#L25

Reply:
@targeral @10Derozan can help confirm this problem?